ui/src/routes/workspaces.tsx

```diff
@@ -1,11 +1,25 @@
 import { createFileRoute, Outlet } from "@tanstack/react-router";
+import { useEffect } from "react";
 
 import { WorkspaceIdWrapper } from "@flow/features/PageWrapper";
 import LeftPanel from "@flow/features/WorkspaceLeftPanel";
 import { TopNavigation } from "@flow/features/WorkspaceTopNavigation";
+import { useUser } from "@flow/lib/gql";
+import i18n from "@flow/lib/i18n/i18n";
 
-export const Route = createFileRoute("/workspaces")({
-  component: () => (
+const WorkspacesComponent = () => {
+  const { useGetMe } = useUser();
+  const { me } = useGetMe();
+  const selectedLanguage =
+    me?.lang && me.lang !== "und" ? me.lang : i18n.language;
+
+  useEffect(() => {
+    if (selectedLanguage) {
+      i18n.changeLanguage(selectedLanguage);
+    }
+  }, [selectedLanguage]);
+
```

Comment:
Is it possible to move this to [here](https://github.com/reearth/reearth-flow/blob/c782baca8b7b74b2adbcff6cab28faa6e183080d/ui/src/lib/i18n/Provider.tsx)?

Reply:
No problem! Will move across now and update.